test/e2e/bootstrappable/topics.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+from dataclasses import dataclass, field
+
+from acktest.bootstrapping import Bootstrappable
+from acktest.resources import random_suffix_name
+import boto3
+
+
+@dataclass
+class Topics(Bootstrappable):
```

Comment:
Probably better to just make this `class Topic` and add 2 of them to your `BootstrapResources` object? `topic1` and `topic2` really don't have much meaning, and there may be a case where you want 3 later?